tests/foreman/cli/test_satellitesync.py

```diff
@@ -1333,6 +1301,184 @@ def test_postive_export_import_cv_with_file_content(
+            {
+                'id': exporting_cv['id'],
+                'organization-id': module_org.id,
+                'repository-id': module_synced_custom_repo['id'],
+            }
+        )
+        target_sat.cli.ContentView.publish({'id': exporting_cv['id']})
+        exporting_cv = target_sat.cli.ContentView.info({'id': exporting_cv['id']})
+        assert len(exporting_cv['versions']) == 1
+        exporting_cvv_id = exporting_cv['versions'][0]['id']
+        # Check exported files
+        exported_packages = target_sat.cli.Package.list(
+            {'content-view-version-id': exporting_cvv_id}
+        )
+        assert exported_packages
+        # Verify export directory is empty
+        assert target_sat.validate_pulp_filepath(module_org, PULP_EXPORT_DIR) == ''
+        # Export the CV
+        export = target_sat.cli.ContentExport.completeVersion(
+            {'id': exporting_cvv_id, 'organization-id': module_org.id}
+        )
+        import_path = target_sat.move_pulp_archive(module_org, export['message'])
+        # Check that files are present in the import_path
+        result = target_sat.execute(f'ls {import_path}')
+        assert result.stdout != ''
+        # Run the import and verify content
+        target_sat.cli.ContentImport.version(
+            {'organization-id': function_import_org.id, 'path': import_path}
+        )
+        importing_cv = target_sat.cli.ContentView.info(
+            {'name': cv_name, 'organization-id': function_import_org.id}
+        )
+        assert len(importing_cv['versions']) == 1
+        imported_packages = target_sat.cli.Package.list(
+            {'content-view-version-id': importing_cv['versions'][0]['id']}
+        )
+        assert imported_packages
+        assert exported_packages == imported_packages
+
+    @pytest.mark.tier2
+    @pytest.mark.parametrize(
+        'function_synced_rhel_repo',
+        ['rhae2'],
+        indirect=True,
+    )
+    def test_positive_export_rerun_failed_import(
+        self,
+        target_sat,
+        config_export_import_settings,
+        export_import_cleanup_function,
+        function_synced_rhel_repo,
+        function_sca_manifest_org,
+        function_import_org_with_manifest,
+    ):
+        """Verify that import can be rerun successfully after failed import.
+
+        :id: 73e7cece-9a93-4203-9c2c-813d5a8d7700
+
+        :parametrized: yes
+
+        :setup:
+            1. Enabled and synced RH repository.
+
+        :steps:
+            1. Create CV, add repo from the setup, publish it and run export.
+            2. Start import of the CV into another organization and kill it before it's done.
+            3. Rerun the import again, let it finish and check the CVV was imported.
+
+        :expectedresults:
+            1. First import should fail, no CVV should be added.
+            2. Second import should succeed without errors and should contain the CVV.
+
+        :CaseImportance: Medium
```

Comment:
This looks like a high-importance case as the possibilities are high for such case in customer env !

Reply:
Here I just followed the severity reported at that BZ, which is medium.